controller/array_action/array_mediator_ds8k.py

```diff
@@ -215,33 +215,28 @@ def _generate_volume_response(self, api_volume):
             array_address=self.service_address,
             copy_source_id=self._get_copy_source_id(api_volume=api_volume),
             pool_name=api_volume.pool,
-            array_type=self.array_type
+            array_type=self.array_type,
+            space_efficiency=api_volume.tp
         )
 
     @staticmethod
-    def get_se_capability_value(capabilities):
-        capability = capabilities.get(config.CAPABILITIES_SPACEEFFICIENCY)
-        if capability:
-            capability = capability.lower()
-            if capability == config.CAPABILITY_THIN:
+    def get_se_space_efficiency_value(space_efficiency):
```

Comment:
`se` and `space_efficiency` essentially mean the same thing.
consider renaming to something like `get_array_space_efficiency`.

btw why is it public?

Reply:
right, I took it out of the class altogether